main.go

```diff
@@ -49,16 +49,26 @@ func exec() int {
 
 	defer logFile.Close()
 
+	log := &logrus.Logger{
+		Out: os.Stdout,
+		Formatter: &logrus.TextFormatter{
+			ForceColors:      true,
+			DisableTimestamp: true,
+		},
+	}
+
+	log.SetLevel(logrus.InfoLevel)
+
 	h, err := os.Hostname()
 	if err != nil {
-		logrus.Debug(err)
+		log.Debug(err)
```

Comment:
after setting `log.SetLevel(logrus.InfoLevel)` we'll never see this debug message

Reply:
Should we rethink how we handle logrus logs in the cmd?